src/client/components/ShoppingCart/ShoppingCart.component.js

```diff
@@ -7,32 +7,97 @@ import RemoveFromShoppingCart from './RemoveFromShoppingCart';
-          <div>
-            <li key={product.id} className="cart-list"> 
-              <div className="cart-product-container">
-                <ShoppingCartImage product={product} />
-                <ModifyProductQuantity
-                  product={product}
-                  shoppingCart={shoppingCart}
-                  setShoppingCart={setShoppingCart}
-                />
-                <RemoveFromShoppingCart
-                  product={product}
-                  shoppingCart={shoppingCart}
-                  setShoppingCart={setShoppingCart}
-                />
+    <div className="parent-div">
+      1.
+      <div className="detail-container">
+        <div>
+          <ul className="cart-container">
+            {shoppingCart.map((product) => (
+              <div>
+                <li key={product.id} className="cart-list">
+                  <div className="cart-product-container">
+                    <ShoppingCartImage product={product} />
+                    <ModifyProductQuantity
+                      product={product}
+                      shoppingCart={shoppingCart}
+                      setShoppingCart={setShoppingCart}
+                      totalPrice={totalPrice}
+                      setTotalPrice={setTotalPrice}
+                    />
+                    <RemoveFromShoppingCart
+                      product={product}
+                      shoppingCart={shoppingCart}
+                      setShoppingCart={setShoppingCart}
+                    />
+                  </div>
+                </li>
+
+                <div className="line" />
               </div>
-            </li>
+            ))}
+          </ul>
+        </div>
+        <div className="right-div">
+          <div className="cart-detail-container">
+            <div>Cart Details</div>
 
-            <div className="line" />
+            <div>
+              Total:<span className="total-price">{totalPriceOfProducts}</span>{' '}
+              DKK
+            </div>
+
+            <div>
+              Discount:<span className="total-discount">{totalDiscount} </span>
+              DKK
+            </div>
+            <div className="line"></div>
+            <div>
+              You Pay:<span className="total-payment">{amountToBePayed}</span>{' '}
+              DKK
+            </div>
+
+            <div>
+              <button className="proceed-payment">Proceed to payment</button>
+            </div>
+          </div>
+          <div>
+            <button className="continue-shop">Continue Shopping</button>
           </div>
-        ))}
-      </ul>
+        </div>
+      </div>
+      2.
+      <div className="user-detail">
+        <div>My Details</div>
+        <div>Name:</div>
+        <div>Email:</div>
+      </div>
+      3.
+      <div className="delivery-info">
+        <div>DELIVERY INFORMATION:</div>
+        <div>Address:</div>
+        <div>City:</div>
+        <div>Country:</div>
+        <div>Zipcode:</div>
```

Comment:
We have working endpoints, I think you should be making use of them to display data!

Reply:
need to do